mmd_tools/converters/mesh_converter.py

```
@@ -0,0 +1,619 @@
+                shader, "specularColor", material.specular[:3], "double3"
+            )
+
+            # スペキュラー係数の取得（PMDとPMXで異なる）
+            specular_coef = None
+            if hasattr(material, "specular_coefficient"):
+                specular_coef = material.specular_coefficient
+            elif hasattr(material, "specular_power"):
+                specular_coef = material.specular_power
+
+            if specular_coef is not None:
+                # スペキュラー係数（MMDの0-100をStandardSurfaceの0-1にマッピング）
```

Ruff (0.12.2)

Local variable `specular_weight` is assigned to but never used

Remove assignment to unused variable `specular_weight`

(F841)